### DISCUSSION TITLE
KAT-V2/manual

### DISCUSSION THREAD

**Discussion by giscus** [wontory/auto-table#7] [Announcements]
# KAT-V2/manual

Kyonggi University Automatic Timetable Generator

http://localhost:3002/KAT-V2/manual

**Comment by bbjbc** [comment]
편하네요~ 앞으로 애용할게요 ~

**Reply by devwontory** [reply]
감사합니다~!

**Comment by SunwooJaeho** [comment]
어머~^^ 세상이.언제 이렇게 발전했는지 다양한.프로그램이. 속속히 나오네요^^ 애용하겠읍니다.^^7

**Reply by devwontory** [reply]
감사합니다~!
